Fix Eslint errors

running the lint script:
```sh
npm run lint
```

gives 12 errors and 199 warnings. Would be good to get these fixed so we can enforce rule that all tests must pass before a PR can be merged.

```sh
> eslint --ext tsx --ext ts src && tsc --noEmit

(node:23724) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "..\..\..\.eslintrc")
(Use `node --trace-deprecation ...` to show where the warning was created)

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\AbstractNavItem.tsx
  46:3  error  Object Literal Property name `aria-controls` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Accordion.tsx
  43:19  warning  Delete `(`  prettier/prettier
  70:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Alert.tsx
   94:15  warning  Delete `(`  prettier/prettier
  145:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Breadcrumb.tsx
  45:32  warning  Delete `(`  prettier/prettier
  69:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Card.tsx
   90:20  warning  Delete `(`  prettier/prettier
  137:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Carousel.tsx
  615:28  warning  Delete `(`  prettier/prettier
  617:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\CarouselItem.tsx
  27:22  warning  Delete `(`  prettier/prettier
  49:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\DataTable.tsx
  146:5   warning  React Hook React.useMemo has a missing dependency: 'columns'. Either include it or remove the dependency array  react-hooks/exhaustive-deps       
  150:32  error    'columns' is already declared in the upper scope                                                                @typescript-eslint/no-shadow      

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Dropdown.tsx
  127:28  warning  Delete `(`  prettier/prettier
  190:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\DropdownMenu.tsx
  167:12  warning  Replace `⏎······menuProps,⏎······{·hasShown,·popper,·show,·alignEnd,·toggle·},` with `menuProps,·{·hasShown,·popper,·show,·alignEnd,·toggle·}]·=` 
 prettier/prettier
  170:5   warning  Replace `]·=` with `·`
 prettier/prettier
  171:1   warning  Replace `······` with `········`
 prettier/prettier
  172:1   warning  Insert `··`
 prettier/prettier
  173:1   warning  Insert `··`
 prettier/prettier
  174:7   warning  Insert `··`
 prettier/prettier
  175:1   warning  Insert `··`
 prettier/prettier
  176:1   warning  Replace `······` with `········`
 prettier/prettier
  177:1   warning  Insert `··`
 prettier/prettier
  178:1   warning  Insert `··`
 prettier/prettier
  179:7   warning  Insert `··`
 prettier/prettier
  180:1   warning  Insert `··`
 prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Figure.tsx
  11:24  warning  Delete `(`  prettier/prettier
  13:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Form.tsx
  69:24  warning  Delete `(`  prettier/prettier
  95:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormCheck.tsx
  142:30  warning  Delete `(`  prettier/prettier
  229:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormControl.tsx
  129:51  warning  Replace `⏎··'input',⏎··FormControlProps⏎>·=` with `'input',·FormControlProps>·=⏎·`  prettier/prettier
  133:1   warning  Replace `··` with `····`                                                            prettier/prettier
  134:1   warning  Insert `··`                                                                         prettier/prettier
  135:1   warning  Insert `··`                                                                         prettier/prettier
  136:7   warning  Insert `··`                                                                         prettier/prettier
  137:1   warning  Insert `··`                                                                         prettier/prettier
  138:7   warning  Insert `··`                                                                         prettier/prettier
  139:1   warning  Insert `··`                                                                         prettier/prettier
  140:1   warning  Replace `······` with `········`                                                    prettier/prettier
  141:1   warning  Insert `··`                                                                         prettier/prettier
  142:1   warning  Replace `······` with `········`                                                    prettier/prettier
  143:1   warning  Insert `··`                                                                         prettier/prettier
  144:1   warning  Replace `······` with `········`                                                    prettier/prettier
  145:7   warning  Insert `··`                                                                         prettier/prettier
  146:1   warning  Replace `······` with `········`                                                    prettier/prettier
  147:1   warning  Insert `··`                                                                         prettier/prettier
  148:7   warning  Insert `··`                                                                         prettier/prettier
  149:1   warning  Insert `··`                                                                         prettier/prettier
  150:5   warning  Insert `··`                                                                         prettier/prettier
  151:1   warning  Replace `····` with `······`                                                        prettier/prettier
  152:1   warning  Insert `··`                                                                         prettier/prettier
  153:1   warning  Insert `··`                                                                         prettier/prettier
  154:1   warning  Insert `··`                                                                         prettier/prettier
  155:7   warning  Insert `··`                                                                         prettier/prettier
  156:1   warning  Insert `··`                                                                         prettier/prettier
  158:1   warning  Replace `····` with `······`                                                        prettier/prettier
  160:5   warning  Insert `··`                                                                         prettier/prettier
  161:1   warning  Replace `····` with `······`                                                        prettier/prettier
  162:1   warning  Insert `··`                                                                         prettier/prettier
  163:1   warning  Replace `····` with `······`                                                        prettier/prettier
  164:1   warning  Insert `··`                                                                         prettier/prettier
  165:1   warning  Replace `····` with `······`                                                        prettier/prettier
  166:1   warning  Insert `··`                                                                         prettier/prettier
  167:1   warning  Insert `··`                                                                         prettier/prettier
  168:7   warning  Insert `··`                                                                         prettier/prettier
  169:1   warning  Insert `··`                                                                         prettier/prettier
  170:9   warning  Insert `··`                                                                         prettier/prettier
  171:7   warning  Insert `··`                                                                         prettier/prettier
  172:1   warning  Insert `··`                                                                         prettier/prettier
  173:7   warning  Insert `··`                                                                         prettier/prettier
  174:1   warning  Replace `········` with `··········`                                                prettier/prettier
  175:1   warning  Insert `··`                                                                         prettier/prettier
  176:7   warning  Insert `··`                                                                         prettier/prettier
  177:1   warning  Replace `····` with `······`                                                        prettier/prettier
  179:1   warning  Insert `··`                                                                         prettier/prettier
  180:7   warning  Insert `··`                                                                         prettier/prettier
  181:1   warning  Insert `··`                                                                         prettier/prettier
  182:5   warning  Insert `··`                                                                         prettier/prettier
  184:1   warning  Insert `··`                                                                         prettier/prettier
  185:7   warning  Insert `··`                                                                         prettier/prettier
  186:1   warning  Insert `··`                                                                         prettier/prettier
  187:1   warning  Replace `········` with `··········`                                                prettier/prettier
  188:1   warning  Insert `··`                                                                         prettier/prettier
  189:1   warning  Replace `········` with `··········`                                                prettier/prettier
  190:1   warning  Insert `··`                                                                         prettier/prettier
  191:1   warning  Replace `········` with `··········`                                                prettier/prettier
  192:1   warning  Insert `··`                                                                         prettier/prettier
  193:1   warning  Replace `··········` with `············`                                            prettier/prettier
  194:11  warning  Insert `··`                                                                         prettier/prettier
  195:1   warning  Insert `··`                                                                         prettier/prettier
  196:11  warning  Insert `··`                                                                         prettier/prettier
  197:1   warning  Insert `··`                                                                         prettier/prettier
  198:7   warning  Insert `··`                                                                         prettier/prettier
  199:1   warning  Insert `··`                                                                         prettier/prettier
  200:3   warning  Insert `··`                                                                         prettier/prettier
  201:1   warning  Insert `··`                                                                         prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFile.tsx
   30:3   error    Type Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE            @typescript-eslint/naming-convention
  114:3   error    Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention
  132:28  warning  Delete `(`                                                                                                             prettier/prettier
  228:13  warning  Delete `)`                                                                                                             prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFileLabel.tsx
  38:3  error  Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\ListGroup.tsx
   63:30  warning  Delete `(`  prettier/prettier
  104:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Media.tsx
  24:22  warning  Delete `(`  prettier/prettier
  40:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Modal.tsx
  220:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Nav.tsx
  124:18  warning  Delete `(`  prettier/prettier
  174:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\NavDropdown.tsx
   66:34  warning  Delete `(`  prettier/prettier
  113:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Navbar.tsx
  154:24  warning  Delete `(`  prettier/prettier
  228:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Overlay.tsx
  23:3  error  Type Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention 

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Pagination.tsx
  47:32  warning  Delete `(`  prettier/prettier
  65:14  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Popover.tsx
   80:26  warning  Delete `(`  prettier/prettier
  117:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Switch.tsx
  12:24  warning  Delete `(`  prettier/prettier
  14:13  warning  Delete `)`  prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Tab.tsx
  12:3  error  'disabled' PropType is defined but prop is never used      react/no-unused-prop-types
  13:3  error  'tabClassName' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TabContainer.tsx
  18:3  error  'defaultActiveKey' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TabPane.tsx
   97:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention
  118:5  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TablePagination.tsx
  148:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps
  151:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps
  154:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Toast.tsx
   63:45  warning  Replace `⏎··'div',⏎··ToastProps⏎>·=` with `'div',·ToastProps>·=⏎·`  prettier/prettier
   67:1   warning  Insert `··`                                                         prettier/prettier
   68:5   warning  Insert `··`                                                         prettier/prettier
   69:1   warning  Insert `··`                                                         prettier/prettier
   70:1   warning  Replace `······` with `········`                                    prettier/prettier
   71:7   warning  Insert `··`                                                         prettier/prettier
   72:1   warning  Replace `······` with `········`                                    prettier/prettier
   73:7   warning  Insert `··`                                                         prettier/prettier
   74:1   warning  Insert `··`                                                         prettier/prettier
   75:7   warning  Insert `··`                                                         prettier/prettier
   76:1   warning  Insert `··`                                                         prettier/prettier
   77:1   warning  Replace `······` with `········`                                    prettier/prettier
   78:1   warning  Insert `··`                                                         prettier/prettier
   79:1   warning  Insert `··`                                                         prettier/prettier
   80:5   warning  Insert `··`                                                         prettier/prettier
   81:1   warning  Insert `··`                                                         prettier/prettier
   82:5   warning  Insert `··`                                                         prettier/prettier
   84:1   warning  Insert `··`                                                         prettier/prettier
   85:1   warning  Replace `····` with `······`                                        prettier/prettier
   86:5   warning  Insert `··`                                                         prettier/prettier
   87:1   warning  Insert `··`                                                         prettier/prettier
   89:5   warning  Insert `··`                                                         prettier/prettier
   90:1   warning  Insert `··`                                                         prettier/prettier
   91:1   warning  Replace `······` with `········`                                    prettier/prettier
   92:5   warning  Insert `··`                                                         prettier/prettier
   94:1   warning  Replace `····` with `······`                                        prettier/prettier
   95:1   warning  Insert `··`                                                         prettier/prettier
   97:1   warning  Insert `··`                                                         prettier/prettier
   98:7   warning  Insert `··`                                                         prettier/prettier
   99:1   warning  Insert `··`                                                         prettier/prettier
  100:7   warning  Insert `··`                                                         prettier/prettier
  101:1   warning  Insert `··`                                                         prettier/prettier
  103:1   warning  Replace `····` with `······`                                        prettier/prettier
  104:1   warning  Insert `··`                                                         prettier/prettier
  105:1   warning  Replace `······` with `········`                                    prettier/prettier
  106:5   warning  Insert `··`                                                         prettier/prettier
  108:1   warning  Insert `··`                                                         prettier/prettier
  109:7   warning  Insert `··`                                                         prettier/prettier
  110:1   warning  Insert `··`                                                         prettier/prettier
  111:1   warning  Replace `······` with `········`                                    prettier/prettier
  112:1   warning  Insert `··`                                                         prettier/prettier
  113:5   warning  Insert `··`                                                         prettier/prettier
  115:1   warning  Insert `··`                                                         prettier/prettier
  117:1   warning  Insert `··`                                                         prettier/prettier
  118:7   warning  Insert `··`                                                         prettier/prettier
  119:1   warning  Insert `··`                                                         prettier/prettier
  120:1   warning  Replace `········` with `··········`                                prettier/prettier
  121:1   warning  Insert `··`                                                         prettier/prettier
  122:1   warning  Insert `··`                                                         prettier/prettier
  123:11  warning  Insert `··`                                                         prettier/prettier
  124:1   warning  Insert `··`                                                         prettier/prettier
  125:9   warning  Insert `··`                                                         prettier/prettier
  126:1   warning  Insert `··`                                                         prettier/prettier
  127:1   warning  Replace `········` with `··········`                                prettier/prettier
  128:9   warning  Insert `··`                                                         prettier/prettier
  129:1   warning  Insert `··`                                                         prettier/prettier
  130:9   warning  Insert `··`                                                         prettier/prettier
  131:7   warning  Insert `··`                                                         prettier/prettier
  132:1   warning  Replace `····` with `······`                                        prettier/prettier
  134:1   warning  Insert `··`                                                         prettier/prettier
  135:1   warning  Insert `··`                                                         prettier/prettier
  136:9   warning  Insert `··`                                                         prettier/prettier
  137:1   warning  Insert `··`                                                         prettier/prettier
  138:13  warning  Insert `··`                                                         prettier/prettier
  139:1   warning  Insert `··`                                                         prettier/prettier
  140:1   warning  Replace `········` with `··········`                                prettier/prettier
  141:1   warning  Insert `··`                                                         prettier/prettier
  142:1   warning  Replace `········` with `··········`                                prettier/prettier
  143:1   warning  Insert `··`                                                         prettier/prettier
  144:5   warning  Insert `··`                                                         prettier/prettier
  145:1   warning  Replace `··` with `····`                                            prettier/prettier
  146:1   warning  Insert `··`                                                         prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\ToggleButtonGroup.tsx
   87:51  warning  Delete `(`                                                                                                                                                                      prettier/prettier
   89:12  warning  Replace `⏎······children,⏎······type,⏎······name,⏎······value,⏎······onChange,⏎······...controlledProps` with `·children,·type,·name,·value,·onChange,·...controlledProps·}·=`  prettier/prettier
   96:5   warning  Replace `}·=` with `·`                                                                                                                                                          prettier/prettier
   97:7   warning  Insert `··`                                                                                                                                                                     prettier/prettier
   98:1   warning  Insert `··`                                                                                                                                                                     prettier/prettier
  148:13  warning  Delete `)`                                                                                                                                                                      prettier/prettier

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\createWithBsPrefix.tsx
  17:39  warning  Insert `,`  prettier/prettier

✖ 211 problems (12 errors, 199 warnings)
  0 errors and 195 warnings potentially fixable with the `--fix` option.
```

I think we need to:
1) downgrade eslint to v7.32.0
2) run `npm install eslint-plugin-react-hooks@latest --save-dev`

Then we get the following:

```bash
(node:25748) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "..\..\..\.eslintrc")
(Use `node --trace-deprecation ...` to show where the warning was created)

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\AbstractNavItem.tsx
  46:3  error  Object Literal Property name `aria-controls` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\DataTable.tsx
  146:5   warning  React Hook React.useMemo has a missing dependency: 'columns'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
  150:32  error    'columns' is already declared in the upper scope                                                                @typescript-eslint/no-shadow

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFile.tsx
   30:3  error  Type Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE            @typescript-eslint/naming-convention
  114:3  error  Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFileLabel.tsx
  38:3  error  Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Modal.tsx
  220:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Overlay.tsx
  23:3  error  Type Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Tab.tsx
  12:3  error  'disabled' PropType is defined but prop is never used      react/no-unused-prop-types
  13:3  error  'tabClassName' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TabContainer.tsx
  18:3  error  'defaultActiveKey' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TabPane.tsx
   97:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention
  118:5  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TablePagination.tsx
  148:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps
  151:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps
  154:8  warning  React Hook useCallback has a missing dependency: 'onPageChange'. Either include it or remove the dependency array. If 'onPageChange' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps

✖ 16 problems (12 errors, 4 warnings)
```

There are 8 remaining issues:

```
C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\AbstractNavItem.tsx
  46:3  error  Object Literal Property name `aria-controls` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFile.tsx
   30:3  error  Type Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE            @typescript-eslint/naming-convention
  114:3  error  Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\FormFileLabel.tsx
  38:3  error  Object Literal Property name `data-browse` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Modal.tsx
  220:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\Overlay.tsx
  23:3  error  Type Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

C:\Users\coliff\Documents\GitHub\modus-react-bootstrap\src\TabPane.tsx
   97:3  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention
  118:5  error  Object Literal Property name `aria-labelledby` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -3,6 +3,7 @@
   "plugins": ["prettier"],
   "rules": {
     "import/extensions": "off",
-    "prettier/prettier": "warn"
+    "prettier/prettier": "warn",
+    "@typescript-eslint/naming-convention": "off"
   }
 }

